<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name="bbox_x" default="0.5"/>
	<arg name="bbox_y" default="0.5"/>
	<arg name="bbox_z" default="0.5"/>
	<arg name="num_of_agents" default="10"/>

	<node name="collision_detector" pkg="ego_planner" type="collision_detector_for_ego_swarm.py" output="screen">
		<param name="bbox_x" value="$(arg bbox_x)"/>
		<param name="bbox_y" value="$(arg bbox_y)"/>
		<param name="bbox_z" value="$(arg bbox_z)"/>
		<param name="num_of_agents" value="$(arg num_of_agents)"/>
	</node>

</launch>



